Core/src/org/sleuthkit/autopsy/discovery/search/DomainSearchCache.java

```diff
@@ -57,7 +57,7 @@
                     groupSortingType, domainSortingMethod, caseDb, centralRepoDb);
 
             return cache.get(searchKey);
-        } catch (ExecutionException ex) {
+        } catch (Throwable ex) {
```

Comment:
Was there an error coming out of this code?

Reply:
Woops forgot to remove that had it in for debugging before I realized the error was because I didnt have a valid grouping criteria created yet.  Will remove that.